src/app/child-dev-project/children/child-block/child-block.component.html

```diff
@@ -1,44 +1,15 @@
 <span
   *ngIf="entity"
-  (mouseenter)="showTooltip()"
-  (mouseleave)="hideTooltip()"
-  [ngClass]="{ inactive: !entity.isActive }"
-  class="truncate-text container"
```

Comment:
Why did you remove this container class? It seems like this creates the vertical alignment of the name and the project number